packages/service-core/src/routes/route-register.ts

```diff
@@ -83,13 +87,12 @@ export function registerFastifyRoutes(
           try {
             await reply.send(response.data);
           } finally {
-            await response.afterSend?.();
-            logger.info(`${e.method} ${request.url}`, {
+            await response.afterSend?.({ clientClosed: request.socket.closed });
+            requestLogger.info(`${e.method} ${request.url}`, {
```

Comment:
Should we be concerned with disposing these child loggers which are created per request? It seems like the Logger has a `close` method which we might be able to call in the `afterSend` hook

Reply:
I couldn't find anything indicating that the child loggers need to be disposed explicitly. They re-use the same transports etc from the parent logger, only keeping some additional log metadata. They will be automatically garbage collected when not used anymore.